### PR TITLE
feat: Include dev info on meroxa version

### PR DESCRIPTION
**Pull request by raulb**
## Description of change

Fixes https://github.com/meroxa/cli/issues/340

## Type of change

- [x]  New feature
- [ ]  Bug fix
- [ ]  Refactor
- [ ]  Documentation

## Demo

**Before**

```
$ make install
$ meroxa version
meroxa/dev darwin/amd64
```

**After**

```
$ make install
$ meroxa version
meroxa/dev:7abf3288 v2.0.0-alpha.3-1-g7abf3288 darwin/amd64
```

If a change was made on top of a commit:

```
$ meroxa version
meroxa/dev:7abf3288 v2.0.0-alpha.3-1-g7abf3288 (updated) darwin/amd64
```

If it was built from same source as a tag:

```
$ meroxa version
meroxa/dev:7abf3288 v2.0.0-alpha.3 darwin/amd64
```

**Note:** Demo was shared in slack if you still need more info.

### CODE REVIEW

File: cmd/meroxa/main.go [hunk 1]
```diff
@@ -16,13 +16,20 @@ limitations under the License.
...
+var (
+	version      = "dev"
+	GitCommit    string
+	GitUntracked string
+	GitLatestTag string
+)
 
 func main() {
-	global.Version = version
+	global.Version = fmt.Sprintf("%s:%s %s %s", version, GitCommit, GitLatestTag, GitUntracked)
```
Comment (jayjayjpg) [inline cmd/meroxa/main.go:33]: This is super neat!

**Review by jayjayjpg — APPROVED** [review]
:sparkles:

**Review by ericcheatham — APPROVED** [review]
🌮

**Review by janelletavares — APPROVED** [review]
this PR is so much smaller than I thought it would be! nice

**Pull request merged**